core/src/consensus.rs

```diff
@@ -62,7 +62,7 @@ pub const COINBASE_MATURITY: u64 = DAY_HEIGHT;
 /// function of block height (time). Starts at 90% losing a percent
 /// approximately every week. Represented as an integer between 0 and 100.
 pub fn secondary_pow_ratio(height: u64) -> u64 {
-	90u64.saturating_sub(height / (2 * YEAR_HEIGHT / 90))
+	90u64.saturating_add(height / (2 * YEAR_HEIGHT / 90))
```

Comment:
Haven't you just replaced the wrong line? The `saturating_sub` on the reward function is still there.

Reply:
Oh wow. Can't touch code anymore. Just fixed, now out to digging a hole for myself.